data/174/641/519/3/1746415193.geojson

```diff
@@ -0,0 +1,148 @@
+{
+  "id": 1746415193,
+  "type": "Feature",
+  "properties": {
+    "date:inception_lower":"1990-01-01",
+    "date:inception_upper":"1990-12-31",
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"1990",
+    "eu-cdda:cddaCountr":"DE",
+    "eu-cdda:cddaId":26064,
```

Comment:
Same deal as other PRs, if this ID is only unique for the layer than the source needs to be explicate to that layer.

Reply:
Same comment from the FR PR... this is a different source than the UK data, but these are unique codes to the whole CDDA database.. so I think we're good here?

Reply:
Agreed